src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -682,3 +682,22 @@ PARAM_DEFINE_INT32(MPC_OBS_AVOID, 0);
  * @group Mission
  */
 PARAM_DEFINE_INT32(MPC_YAW_MODE, 0);
+
+/**
+ * Flag to enable the use of a range sensor for collision avoidance.
+ *
+ * @boolean
+ * @group Multicopter Position Control
+ */
+PARAM_DEFINE_INT32(MPC_COL_AVOID, 0);
+/**
+ * Minimum Obstacle Distance at which the vehicle should not get closer
```

Comment:
"Obstacle Distance" above are not proper nouns, so should not be capitalised.

Maybe change to:

>Minimum allowed target obstacle distance